package.json

```diff
@@ -22,7 +22,8 @@
   "devDependencies": {
     "eslint": "^7.1.0",
     "esm": "^3.2.25",
-    "jasmine": "^3.5.0"
+    "jasmine": "^3.5.0",
+    "rollup": "^2.10.9"
```

Comment:
In my local env I'm using a really old version of `rollup` (`v0.68.1`).
The `npm run build` script may be working well only with my current version.
Anyways, let me handle the deploy on my end. I'll upgrade `rollup` and do the `2.0.0` version.

Please, rollback the changes that you did in `build.js`, so then I can merge this PR.